test/cmd/images.sh

```diff
@@ -20,125 +21,126 @@ USE_IMAGES=${USE_IMAGES:-$defaultimage}
+os::cmd::expect_success 'oc tag --source=istag mysql:latest tagtest:tag5 --alias'
+os::cmd::expect_success_and_text "oc get is/tagtest --template='{{(index .spec.tags 4).from.kind}}'" 'ImageStreamTag'
 
-oc tag --source=imagestreamimage mysql@${name} tagtest:tag6 --alias
-[ "$(oc get is/tagtest --template='{{(index .spec.tags 5).from.kind}}')" == "ImageStreamImage" ]
+os::cmd::expect_success 'oc tag --source=imagestreamimage mysql@${name} tagtest:tag6 --alias'
+os::cmd::expect_success_and_text "oc get is/tagtest --template='{{(index .spec.tags 5).from.kind}}'" 'ImageStreamImage'
 
-oc tag --source=isimage mysql@${name} tagtest:tag7 --alias
-[ "$(oc get is/tagtest --template='{{(index .spec.tags 6).from.kind}}')" == "ImageStreamImage" ]
+os::cmd::expect_success 'oc tag --source=isimage mysql@${name} tagtest:tag7 --alias'
+os::cmd::expect_success_and_text "oc get is/tagtest --template='{{(index .spec.tags 6).from.kind}}'" 'ImageStreamImage'
 
-oc tag --source=docker mysql:latest tagtest:tag8 --alias
-[ "$(oc get is/tagtest --template='{{(index .spec.tags 7).from.kind}}')" == "DockerImage" ]
+os::cmd::expect_success 'oc tag --source=docker mysql:latest tagtest:tag8 --alias'
+os::cmd::expect_success_and_text "oc get is/tagtest --template='{{(index .spec.tags 7).from.kind}}'" 'DockerImage'
 
-oc tag mysql:latest tagtest:zzz tagtest2:zzz --alias
-[ "$(oc get is/tagtest --template='{{(index .spec.tags 8).from.kind}}')" == "ImageStreamTag" ]
-[ "$(oc get is/tagtest2 --template='{{(index .spec.tags 0).from.kind}}')" == "ImageStreamTag" ]
+os::cmd::expect_success 'oc tag mysql:latest tagtest:zzz tagtest2:zzz --alias'
+os::cmd::expect_success_and_text "oc get is/tagtest --template='{{(index .spec.tags 8).from.kind}}'" 'ImageStreamTag'
+os::cmd::expect_success_and_text "oc get is/tagtest2 --template='{{(index .spec.tags 0).from.kind}}'" 'ImageStreamTag'
 
-oc tag registry-1.docker.io/openshift/origin:v1.0.4 newrepo:latest
-[ "$(oc get is/newrepo --template='{{(index .spec.tags 0).from.kind}}')" == "DockerImage" ]
-oc tag mysql:5.5 newrepo:latest
-[ "$(oc get is/newrepo --template='{{(index .spec.tags 0).from.kind}}')" == "ImageStreamImage" ]
-oc tag mysql:5.5 newrepo:latest --alias
-[ "$(oc get is/newrepo --template='{{(index .spec.tags 0).from.kind}}')" == "ImageStreamTag" ]
+os::cmd::expect_success 'oc tag registry-1.docker.io/openshift/origin:v1.0.4 newrepo:latest'
+os::cmd::expect_success_and_text "oc get is/newrepo --template='{{(index .spec.tags 0).from.kind}}'" 'DockerImage'
+os::cmd::expect_success 'oc tag mysql:5.5 newrepo:latest'
+os::cmd::expect_success_and_text "oc get is/newrepo --template='{{(index .spec.tags 0).from.kind}}'" 'ImageStreamImage'
+os::cmd::expect_success 'oc tag mysql:5.5 newrepo:latest --alias'
+os::cmd::expect_success_and_text "oc get is/newrepo --template='{{(index .spec.tags 0).from.kind}}'" 'ImageStreamTag'
 
 # test creating streams that don't exist
```

Comment:
It creates tags, but not IS. And yes, those gets are prerequisites to the actual tagging operation checks.